Added initial schedule

rmw opened the pull request:
<img width="1374" alt="screen shot 2017-06-15 at 12 48 12 am" src="https://user-images.githubusercontent.com/363764/27167068-7235e5ce-5164-11e7-87da-7da1e4e64e30.png">

Commit:
Added initial schedule

jarmstrng commented:
Hm, I guess I thought @lateesha meant to have one big subnav for each track right beneath the days subnav, instead of within each time block

Unfortunately, the subnav-within-the-time-block doesn't work on small screens

![screen shot 2017-06-15 at 6 41 51 am](https://user-images.githubusercontent.com/7620750/27177941-36fde7c2-5196-11e7-99fe-836bb6988bbb.png)

Commit:
Update schedule header to be navy

deweydell committed:
Tweak design of schedule section

- Make talk titles non-uppercase, more legible
- Tweak spacing on mobile / tablet / desktop so tracks have more space
- Add functionality for multiple speaker labels
